### DISCUSSION TITLE
Allow admins to edit playlists stored in the filesystem

### DISCUSSION THREAD

**Discussion by nadiamoe** [jellyfin/jellyfin#11744] [General]
Since version 10.9, Jellyfin no longer allows users to edit (i.e. add/remove items from) playlists stored as m3u files on the filesystem.

I think that conceptually this is a change in the right direction, as previously any user could edit these playlists, which probably was not ideal from the access control standpoint.

However, I personally did use this feature a lot. For me, it was an easy way to create "collaborative" playlists, that is, playlists that more than one person can edit. It additionally allowed me to store my playlists on a standard format that is not dependent on jellyfin specifically, which is pretty handy as I can leverage other tools to work with playlists this way (such as my own [m3ucat](https://github.com/roobre/m3ucat)).

As it was mentioned [here](https://github.com/jellyfin/jellyfin/issues/11741), it might be possible to bring back local playlist edition to admin users. To me this seems like a reasonable compromise, so I'd like to show my support for this idea!

**Comment by fuzzysh** [comment]
seconded. used this feature often. very frustrating to have it taken away.

**Comment by uranw0lle** [comment]
I need this feature back as well. I currently stick to 10.8 because of it.

**Comment by LukaHummel** [comment]
Seems like a rather strict solution to just disallow editing of 'local' playlists as a whole. Admins should probably be allowed to edit 'local' playlists or assign an owner.

I migrated my music library from YouTube to Jellyfin and used [yt-dlp](https://github.com/yt-dlp/yt-dlp) with [my own little config](https://gist.github.com/LukaHummel/dba323ed4e1c8ec630b618180fcc7aac) to automatically create .m3u files for any YouTube Playlist I was downloading. Now it seems like I still need to manually recreate the playlists to be able to edit them in the future. 😞
